manifests/r/RStudio/RStudio/1.4.1717/RStudio.RStudio.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: RStudio.RStudio
+PackageVersion: 1.4.1717
+PackageLocale: en-US
+Publisher: RStudio
+PublisherUrl: https://www.rstudio.com/
+PublisherSupportUrl: https://www.rstudio.com/support/
+PrivacyUrl: https://www.rstudio.com/about/privacy-policy/
+PackageName: RStudio Desktop
```

Comment:
```suggestion
PackageName: RStudio
```

Reply:
Thanks. Is this a new spec ? 
I used the same as past version
https://github.com/microsoft/winget-pkgs/blob/9d4d34ba1566bafa051c1d0096f7507b68db6720/manifests/r/RStudio/RStudio/1.4.1106/RStudio.RStudio.yaml#L3

Should we update the old one too ?

Reply:
It's to match the naming with what shows in Control Panel's Add or Remove Programs (ARP) so that users can upgrade the RStudio application by doing `winget upgrade RStudio.RStudio`.

Yes, because some people may want to downgrade the application to an older version and may want to upgrade at a later time.